app/policies/supervisor_volunteer_policy.rb

```diff
@@ -0,0 +1,9 @@
+class SupervisorVolunteerPolicy < Struct.new(:user, :record)
+  def create?
+    user.casa_admin? || user.supervisor? || user.volunteer?
```

Comment:
This seems to be used in a few places. You could extract to method. I think this represents anyone that is a logged in user, but I don't enough about the domain and context to say for sure.

Reply:
@noahc I have that on my list after this gets merged it to come back to. Since these headless policies don't inherit from the `ApplicationPolicy` I couldn't reuse those methods here. I plan to extract those next and use them in all the headless policies.

Reply:
Since I went commit by commit, I didn't see that it actually already exists. But obviously, you can't use it.